src/libraries/SwapMath.sol

```diff
@@ -96,12 +97,12 @@ library SwapMath {
 
             if (exactIn && sqrtRatioNextX96 != sqrtRatioTargetX96) {
```

Comment:
if exactIn && sqrtRatioNextX96 != sqrtRatioTargetX96, the whole fee is going to LP and nothing to protocol, which is why some tests are failing. How should protocol fee be calculated here?